validation/tests.json

```diff
@@ -525,18 +525,12 @@
             { "test": "ttml2-valid-content-profile-multiple" },
             { "test": "ttml2-valid-content-profiles-quantified-all-multiple" },
             { "test": "ttml2-valid-content-profiles-unquantified-multiple" },
-            { "test": "ttml2-valid-infer-processor-profile-source-combined" },
```

Comment:
Seems strange that a bunch of tests are being removed from this file but are not being removed by this pull request. Is that intentional? The referenced tests still exist in the repository, and seem worth keeping.

Reply:
They were removed here because they don't test the feature they are listed under. These tests are still referenced elsewhere, e.g.,  under the #profile-full-version-2.

Reply:
Ah, thanks for explaining.